devices/devs.c

```diff
@@ -122,7 +122,7 @@ int devs_sync(unsigned int major, unsigned int minor)
 }
 
 
-int devs_map(unsigned int major, unsigned int minor, addr_t addr, size_t sz, int mode, addr_t memaddr, size_t memsz, int memmode, addr_t *a)
+int devs_map(unsigned int major, unsigned int minor, addr_t addr, size_t sz, int mode, addr_t memaddr, size_t memsz, int memmode)
```

Comment:
I think that `*a` argument is required because it provides information when data described by device address is located in memory.